ConwaysGameOfLife.Tests/UniverseHelper.cs

```diff
@@ -33,35 +33,35 @@ public static bool UniversesAreEqual(Universe obj1, Universe obj2)
             return true;
         }
 
-        public static bool CellsAreEqual(Cell obj1, Cell obj2)
+        public static bool CellsAreEqual(Cell cell1, Cell cell2)
         {
-            var isLocationSame = (obj1.Location.X == obj2.Location.X) && (obj1.Location.Y == obj2.Location.Y);
-            var isCellStateSame = obj1.CellState == obj2.CellState;
+            var isLocationSame = (cell1.Location.X == cell2.Location.X) && (cell1.Location.Y == cell2.Location.Y);
+            var isCellStateSame = cell1.CellState == cell2.CellState;
             return isCellStateSame && isLocationSame;
         }
 
-        public static bool ListsOfCellsAreEqual(List<Cell> obj1, List<Cell> obj2)
+        public static bool ListsOfLocationsAreEqual(List<Location> location1, List<Location> location2)

```

Comment:
suggestion: use plural for a variable associated with a collection (list)

Reply:
either `locations1` or `locationList1` works